@ornikar/jest-config/bin/test-coverage.sh

```diff
@@ -11,4 +11,4 @@ else
   yarn test --collectCoverage --watchAll=false --findRelatedTests $1 --collectCoverageOnlyFrom="$1" --passWithNoTests
 fi
 
-open test-coverage/lcov-report/index.html
+open coverage/lcov-report/index.html
```

Comment:
currently configured here: https://github.com/ornikar/shared-apps/blob/feb45b59b430c831e639bbdc78a2dcad92761f71/%40ornikar/webapp-configs/src/craco/index.ts#L45

so it's a breaking ?

Reply:
It seems to be jest default output folder. Don't know why we overrode it in `webapp-configs` ?